tests/core/test_dump_restore.py

```diff
@@ -1,141 +1,43 @@
-    definition_period = MONTH
-
-    def formula(person, period):
-        return person('intermediate', period)
-
-
-def create_filled_tax_benefit_system():
-    tax_benefit_system = CountryTaxBenefitSystem()
-    tax_benefit_system.add_variables(input, intermediate, other_input, output)
-
-    return tax_benefit_system
-
-
-# TaxBenefitSystem instance declared after formulas
-
-
-tax_benefit_system = create_filled_tax_benefit_system()
-
-
-month = '2018-08'
-scenario = tax_benefit_system.new_scenario().init_from_attributes(
-    period = month,
-    input_variables = {
-        'input': 1,
-        },
-    )
-
-
-def test_dump_restore_same_simulation():
-    simulation = scenario.new_simulation()
-    simulation.calculate('output', period = month)
-    assert(simulation.get_holder('output').get_array(month) == simulation.get_holder('input').get_array(month))
-
-    simulation.dump()
-
-    assert(simulation.get_holder('output').get_array(month) == simulation.get_holder('input').get_array(month))
-
-    simulation.get_holder('output').set_input(month, [2])
-
-    next_month = '2018-09'
-    next_value = 3
-    simulation.get_holder('input').set_input(next_month, [next_value])
-    simulation.calculate('output', period = next_month)
-    assert(simulation.get_holder('output').get_array(next_month) == simulation.get_holder('input').get_array(next_month))
-
-    simulation.restore()
-
-    assert(simulation.get_holder('output').get_array(month) == simulation.get_holder('input').get_array(month))
-
-    assert(simulation.get_holder('input').get_array(next_month) is None)
-    assert(simulation.get_holder('intermediate').get_array(next_month) is None)
-    assert(simulation.get_holder('output').get_array(next_month) is None)
-
-
-def test_dump_restore_different_simulations():
-    data_storage_dir = tempfile.mkdtemp(prefix = "openfisca_")
-
-    simulation1 = scenario.new_simulation(data_storage_dir = data_storage_dir)
-    simulation1.calculate('output', period = month, extra_params = [-1, -2])
-
-    simulation1.dump()
-
-    # New simulation with the same storage directory
-    simulation2 = scenario.new_simulation(data_storage_dir = data_storage_dir)
-
-    simulation2.restore()
-
-    assert(simulation2.get_holder('input').get_array(month) == np.asarray([1]))
-    assert(simulation2.get_holder('intermediate').get_array(month) == np.asarray([1]))
-    assert(simulation2.get_holder('output').get_array(month, extra_params = [-1, -2]) == np.asarray([1]))
-
-    shutil.rmtree(data_storage_dir)
+from numpy.testing import assert_array_equal
 
+from openfisca_core.simulations import Simulation
+from openfisca_country_template.situation_examples import couple
+from openfisca_core.tools.simulation_dumper import dump_simulation, restore_simulation
 
-def test_dump_restore_different_simulations_different_variables():
-    data_storage_dir = tempfile.mkdtemp(prefix = "openfisca_")
+from .test_countries import tax_benefit_system
```

Comment:
Use absolute import?

Reply:
We can't use absolute imports as the `tests` are not packaged in `openfisca_core`.